google-cloud-tools-plugin/google-account/testSrc/com/google/cloud/tools/intellij/login/integration/PluginXmlTest.java

```diff
@@ -32,7 +31,7 @@
     public void testWellFormed() throws SAXException, IOException {
         XMLReader reader = XMLReaderFactory.createXMLReader();
```

Comment:
nit: Since the line you touched was correctly updated to +2 spaces for indentation (instead of +4), can you fix the rest of the method here for consistency?

Reply:
done